FireRender.Maya.Src/FireRenderObjects.cpp

```diff
@@ -354,7 +357,11 @@ void FireRenderObject::Freshen()
 {
```

Comment:
please add a flag if calculate hash or not instead of accessing context fields here